lang/go/bindings/examples/distributor/pkg/distributor/cmd.go

```diff
@@ -0,0 +1,115 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ * Copyright(c) 2022-2023 Intel Corporation
+ */
+
+package distributor
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/CloudNativeDataPlane/cndp/lang/go/bindings/cne"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+)
+
+const (
+	version    = "v0.0.1"
+	numWorkers = 4
+)
+
+var (
+	config  string
+	verbose bool
+)
+
+func NewDistributorCommand(ctx context.Context) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:    "distributor -c config.jsonc",
+		Short:  "distributor is an example to distribute packets",
```

Comment:
Update the short description to something like this:
'An example to distribute packets via lockless rings between worker threads'

Reply:
Updated